sympy/concrete/expr_with_intlimits.py

```diff
@@ -113,18 +116,18 @@ def change_index(self, var, trafo, newvar=None):
                 beta = p.coeff_monomial(S.One)
                 if alpha.is_number:
                     if alpha == S.One:
-                        limits.append((newvar, alpha*limit[1] + beta, alpha*limit[2] + beta))
+                        limits.append((newvar, alpha * limit[1] + beta, alpha * limit[2] + beta))
                     elif alpha == S.NegativeOne:
-                        limits.append((newvar, alpha*limit[2] + beta, alpha*limit[1] + beta))
+                        limits.append((newvar, alpha * limit[2] + beta, alpha * limit[1] + beta))
```

Comment:
See above about spaces about operators. (More similar places below, won't annotate each one of them.)